zipkin-server/storage-cassandra/src/main/java/zipkin/server/storage/cassandra/dao/EmptyDAO.java

```diff
@@ -0,0 +1,489 @@
+import org.apache.skywalking.oap.server.core.storage.query.IAlarmQueryDAO;
+import org.apache.skywalking.oap.server.core.storage.query.IBrowserLogQueryDAO;
+import org.apache.skywalking.oap.server.core.storage.query.IEventQueryDAO;
+import org.apache.skywalking.oap.server.core.storage.query.ILogQueryDAO;
+import org.apache.skywalking.oap.server.core.storage.query.IMetadataQueryDAO;
+import org.apache.skywalking.oap.server.core.storage.query.IMetricsQueryDAO;
+import org.apache.skywalking.oap.server.core.storage.query.IRecordsQueryDAO;
+import org.apache.skywalking.oap.server.core.storage.query.ISpanAttachedEventQueryDAO;
+import org.apache.skywalking.oap.server.core.storage.query.ITagAutoCompleteQueryDAO;
+import org.apache.skywalking.oap.server.core.storage.query.ITopologyQueryDAO;
+import org.apache.skywalking.oap.server.core.storage.query.ITraceQueryDAO;
+import org.apache.skywalking.oap.server.library.client.request.InsertRequest;
+import org.apache.skywalking.oap.server.library.client.request.UpdateRequest;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+public class EmptyDAO implements
+    INetworkAddressAliasDAO,
+    ITopologyQueryDAO,
+    IMetricsDAO,
+    ILogQueryDAO,
+    ITraceQueryDAO,
+    IMetricsQueryDAO,
+    IAggregationQueryDAO,
+    IAlarmQueryDAO,
+    IRecordsQueryDAO,
+    IRecordDAO,
+    IBrowserLogQueryDAO,
+    IMetadataQueryDAO,
+    IProfileTaskQueryDAO,
+    IProfileTaskLogQueryDAO,
+    IProfileThreadSnapshotQueryDAO,
+    UITemplateManagementDAO,
+    UIMenuManagementDAO,
+    IEventQueryDAO,
+    IEBPFProfilingTaskDAO,
+    IEBPFProfilingScheduleDAO,
+    IEBPFProfilingDataDAO,
+    IContinuousProfilingPolicyDAO,
+    IServiceLabelDAO,
+    ITagAutoCompleteQueryDAO,
+    ISpanAttachedEventQueryDAO
+{
+  @Override
+  public List<NetworkAddressAlias> loadLastUpdate(long timeBucket) {
+    return null;
+  }
+
+  @Override
+  public InsertRequest prepareBatchInsert(Model model, Record record) throws IOException {
+    return null;
+  }
+
+  @Override
+  public UIMenu getMenu(String id) throws IOException {
+    return null;
+  }
+
+  @Override
+  public void saveMenu(UIMenu menu) throws IOException {
+
+  }
+
+  @Override
+  public DashboardConfiguration getTemplate(String id) throws IOException {
+    return null;
+  }
+
+  @Override
+  public List<DashboardConfiguration> getAllTemplates(Boolean includingDisabled) throws IOException {
+    return null;
+  }
+
+  @Override
+  public TemplateChangeStatus addTemplate(DashboardSetting setting) throws IOException {
+    return null;
+  }
+
+  @Override
+  public TemplateChangeStatus changeTemplate(DashboardSetting setting) throws IOException {
+    return null;
+  }
+
+  @Override
+  public TemplateChangeStatus disableTemplate(String id) throws IOException {
+    return null;
+  }
+
+  @Override
+  public void savePolicy(ContinuousProfilingPolicy policy) throws IOException {
+
```

Comment:
Could you create a NotImplementedException? And throw from here?
It is better to see a clear exception rather than NPE or other strange behavior as there is nothing executed.
After all, these methods should not be called.